addon/helpers/deprecated/ago.js

```diff
@@ -1,6 +1,7 @@
 import Ember from 'ember';
-import { helperFactory } from '../moment-from-now';
+import momentFromNowHelper from '../moment-from-now';
```

Comment:
This breaks 1.10.0.  Causes the Ember.deprecate to immediately fire even when not using {{ago}}